modules/ROOT/pages/upgrade.adoc

```diff
@@ -90,6 +94,13 @@ Before accessing Anypoint Platform PCE, ensure that you configure your NSF serve
 .. Select *Save*.
 .. Select *Test NFS Server* to test that your NFS server is configured correctly.
 
+. Unless you are updating Anypoint Platform PCE from 2.0.x to 2.1.x, configure the following items:
```

Comment:
...and what happens if you are upgrading from 2.0 to 2.1? Are we again referring to prereqs in such a way that interferes with the main upgrade task?
If this is really a problem, maybe we should consider SEPARATE upgrade tasks for each situation?